cwms-data-api/src/main/java/cwms/cda/api/LocationGroupController.java

```diff
@@ -135,15 +135,20 @@ Boolean.class, false, metrics, name(LocationGroupController.class.getName(),
     @OpenApi(
         pathParams = {
             @OpenApiParam(name = GROUP_ID, required = true, description = "Specifies "
-                    + "the location_group whose data is to be included in the response")
+                + "the location_group whose data is to be included in the response")
         },
         queryParams = {
             @OpenApiParam(name = OFFICE, required = true, description = "Specifies the "
-                    + "owning office of the location group whose data is to be included "
-                    + "in the response."),
+                + "owning office of the location group whose data is to be included "
+                + "in the response."),
+            @OpenApiParam(name = GROUP_OFFICE_ID, required = true, description = "Specifies the "
```

Comment:
I could use some input on the language for describing the different office parameters.

Reply:
That looks fine to me